AmplifyPlugins/DataStore/Sources/AWSDataStorePlugin/Storage/SQLite/SQLStatement+Condition.swift

```diff
@@ -31,9 +31,9 @@ private func translateQueryPredicate(from modelSchema: ModelSchema,
         if let operation = pred as? QueryPredicateOperation {
             let column = resolveColumn(operation)
```

Comment:
Since we are in SQL-injection territory. Do we know for certain that this column value is safe? (e.g. an identifier without spaces or escape characters).

Reply:
Excellent, thank you.